lib/app.go

```diff
@@ -95,11 +96,6 @@ func newApp(ra *schema.RuntimeApp, podManifest *schema.PodManifest, pod *pkgPod.
 		})
 	}
 
-	// Generate annotations.
-	for _, anno := range ra.Annotations {
-		app.Annotations[anno.Name.String()] = anno.Value
```

Comment:
Ok, thanks for digging through history. I think we should double-check this later and verify that we are not messing between internal annotations and user annotations at any point, but from a quick glance this should be fine.